x-pack/test/api_integration/apis/secops/hosts.ts

```diff
@@ -43,10 +46,10 @@ const hostsTests: KbnTestProvider = ({ getService }) => {
         });
     });
 
-    it('Make sure that pagination is working in Hosts query', () => {
+    it('Make sure that pagination is working in Hosts Table query', () => {
```

Comment:
I'd use `test(` instead of `it(` since everywhere else we're using `test(`

Reply:
Looks like the integration tests need to be `it(` as per that build failure -- so I went ahead and reverted them back. Will use `test(` going forward for unit tests though!